qa/suites/fs/verify/validater/valgrind.yaml

```diff
@@ -1,3 +1,5 @@
+# see http://tracker.ceph.com/issues/20360 and http://tracker.ceph.com/issues/18126
+os_type: centos
```

Comment:
last time this caused issues since it needed 'os_version: 7' as well, or it defaulted to 14.04